### ISSUE TITLE
Return route options from operationResolver

### ISSUE THREAD

**Issue by ivan-tymoshenko** [seriousme/fastify-openapi-glue#483]
First of all, thanks for your work. Great plugin!

My suggestion is to add a _second_ possible return type for `operationResolver` to pass a route options object when it's needed.

Example:

```js
{
  operationResolver: (operationId, method, openApiPath) => {
    return {
      logLevel: 'warn',
      version: '1.1.0',
      handler: async (req, res) => {}
    }
  }
}
```

Tell me if those changes are welcome, I will open a PR.

**Comment by seriousme** [comment]
Hi,

sounds like a good addition !
Happy to receive a PR :-)

Kind regards,
Hans

**Comment by seriousme** [comment]
Thanks for your contribution!
Released as 4.2.0 on NPM.

Kind regards,
Hans
